{
  "contest": 343,
  "handle": "t0x1c",
  "risk": "2",
  "title": "Banned address can still `_invokeMessageCall()` by calling `retryMessage()`",
  "issueId": 48,
  "issueUrl": "https://github.com/code-423n4/2024-03-taiko-findings/issues/48"
}